### PR TITLE
Forecast page changes

### PR DESCRIPTION
**Pull request by floatplane**
- bugfix: disable touch on avalanche problem images: the component isn't fully baked yet
- bugfix: hide divider on collapsible cards, allow touch anywhere in header to toggle
- cleanup: get configuration for RenderHTML into a single component called `HTMLRenderer`, start using preferred fonts

### CODE REVIEW

File: components/AvalancheProblemImage.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import {Text, Image, StyleSheet, TouchableHighlight, useWindowDimensions, View, ActivityIndicator} from 'react-native';
+import {Text, Image, StyleSheet, /*TouchableHighlight,*/ useWindowDimensions, View, ActivityIndicator} from 'react-native';
```
Comment (stevekuznetsov) [inline components/AvalancheProblemImage.tsx:3]: Feel free to just delete the thing - `git revert` is easy and won't leave comments in here.

**Comment by stevekuznetsov** [comment]
When the card is collapsed, the divider is not shown but there's a bunch of white-space left at the bottom of it - it looks like more space than we have on top of the header, I think? Otherwise this looks great!

**Comment by floatplane** [comment]
> When the card is collapsed, the divider is not shown but there's a bunch of white-space left at the bottom of it - it looks like more space than we have on top of the header, I think? Otherwise this looks great!

fixed!

**Comment by floatplane** [comment]
Long press to show HTML source (only when `__DEV__` is true):

Simulator Screen Recording - iPhone 14 Pro Max - 2023-01-05 at 20 04 34

**Pull request merged**